app/routes/object_storage/objects.py

```diff
@@ -157,6 +166,7 @@ def get_object_route(namespace_name, bucket_name, subpath):
         content_type=_object["content_type"],
         response=content,
         headers={
+            "etag": str(uuid.uuid4()),
```

Comment:
After doing the changes described on the upper comment you can simply return `_object["etag"]`